docs/src/ComponentsPage.js

```diff
@@ -445,6 +445,19 @@ const ComponentsPage = React.createClass({
                   <ReactPlayground codeText={Samples.PagerDisabled} />
                 </div>
 
+                {/* Pagination */}
+                <div className='bs-docs-section'>
+                  <h1 id='pagination' className='page-header'>Pagination <small>Pagination</small></h1>
+                  <h2 id='pagination-examples'>Example pagination</h2>
+
+                  <p>Provide pagination links for your site or app with the multi-page pagination component.</p>
+                  <ReactPlayground codeText={Samples.PaginationBasic} />
+
+                  <p>More controller such as first, last, previous, next and ellipsis.</p>
```

Comment:
controller => controllers ? or handlers ? or buttons ?
@mtscout6  What would you suggest ?